src/scalarstats.jl

```diff
@@ -166,16 +206,40 @@ nquantile{T<:Real}(v::AbstractArray{T}, n::Integer) = quantile(v, (0:n)/n)
 #############################
 
 # span, i.e. the range minimum(x):maximum(x)
+"""
+    span(x) -> Range
+
+Return the span of an integer array, i.e. the range `minimum(x):maximum(x)`.
+"""
 span{T<:Integer}(x::AbstractArray{T}) = ((a, b) = extrema(x); a:b)
 
 # Variation coefficient: std / mean
+"""
+    variation(x[, m]) -> Float
```

Comment:
Indeed, in many cases we could as well omit it. I guess it's not really needed when a `Float64` is returned. There might be other cases too, like when an array result is clearly expected, or when an integer is the only meaningful return value. Up to you.